src/dist/replication/lib/replica_stub.cpp

```diff
@@ -2201,6 +2206,31 @@ void replica_stub::open_service()
             return result;
         });
 #endif
+    _max_concurrent_bulk_load_downloading_count_command =
+        dsn::command_manager::instance().register_app_command(
+            {"max-concurrent-bulk-load-downloading-count"},
+            "max-concurrent-bulk-load-downloading-count [num | DEFAULT]",
+            "control stub max_concurrent_bulk_load_downloading_count",
+            [this](const std::vector<std::string> &args) {
+                std::string result("OK");
+                if (args.empty()) {
```

Comment:
These if..else.. are too complex, better to refactor like https://github.com/XiaoMi/rdsn/pull/461/files#diff-a6778028d5f3348c9fb2e6cbd55891adR525